docs/openapi/components/schemas/common/LEIaddress.yml

```diff
@@ -75,20 +82,19 @@ properties:
     type: string
     $linkedData:
       term: postalCode
-      '@id': https://schema.org/postalCode
+      '@id': https://www.gleif.org/ontology/Base/hasPostalCode
 additionalProperties: false
 required:
   - type
-  - addressLines
+  - firstAddressLine
   - city
-  - region
   - country
-  - postalCode
 example: |-
   {
     "type": ["LEIaddress"],
     "language": "ru",
```

Comment:
Nothing I can see is in Cyrillic, never mind [Russian](https://localizely.com/language-code/ru/).

Reply:
fixed